packages/gamut-labs/src/landingPage/PageHeroMedia.tsx

```diff
@@ -0,0 +1,45 @@
+import { Column, Video } from '@codecademy/gamut';
+import { breakpoints } from '@codecademy/gamut-styles';
+import styled from '@emotion/styled';
+import { omit } from 'lodash';
+import React from 'react';
+
+import { MediaProps } from './PageHero';
+
+const RightColumn = styled(Column)`
+  margin-left: 1rem;
+
+  @media only screen and (max-width: ${breakpoints.sm}) {
```

Comment:
Oh and we have a thing for this: https://gamut.codecademy.com/storybook/?path=/docs/foundations-layout--page#usage-typescript

```suggestion
  ${mediaQueries.sm} {
```

Reply:
just a heads up, `mediaQueries.sm` uses `min-width` instead of `max-width` so you would need to reverse the styling

Reply:
is this a blocker?

Reply:
Not IMO :)

Reply:
and oh didn't notice the change, good call @christian-dinh !